### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 - Rodrigo Lopes Does Santos and Urs Behrmann

### DISCUSSION THREAD

**Discussion by UBehrmann** [heig-vd-dai-course/heig-vd-dai-course#336] [Show and tell]
Faire une recherche de suite de charactère dans un ou plusieur fichier qui ont des formats différents.

On input la chaîne que l'on cherche et les fichiers cibles. 

On ouput une liste des fichiers dans lesquelles elle a été trouvé et dans quelle format.

**Comment by GeraudSilvestri** [comment]
C'est bon pour moi

**Reply by ludelafo** [reply]
All good for me too!

**Comment by UBehrmann** [comment]
Link to repo: https://github.com/UBehrmann/DAI_Practical_work_1

Put a tag "rendu" on the last commit : b155790858e8c404d124303636ddae034ec9b9b9
